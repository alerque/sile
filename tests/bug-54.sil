\begin[papersize=a6]{document}
\nofolios
\font[family=Gentium Plus,size=12pt]
\set[parameter=document.parindent,value=10mm]
\set[parameter=current.parindent,value=10mm]
First paragraph. \par
Second paragraph.

Third paragraph.
\end{document}
