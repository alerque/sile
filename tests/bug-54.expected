Set paper size 	297.6377985	419.5275636
Begin page
Mx 	43.2283
My 	30.0584
Set font 	Gentium Plus;12;400;;normal;;LTR
T	41 76 85 86 87	(First)
Mx 	68.9264
T	83 68 85 68 74 85 68 83 75	(paragraph)
Mx 	120.1432
T	17	(.)
Mx 	43.2283
My 	44.4584
T	54 72 70 82 81 71	(Second)
Mx 	81.8153
T	83 68 85 68 74 85 68 83 75	(paragraph)
Mx 	133.0321
T	17	(.)
Mx 	43.2283
My 	58.8584
T	55 75 76 85 71	(Third)
Mx 	73.8945
T	83 68 85 68 74 85 68 83 75	(paragraph)
Mx 	125.1113
T	17	(.)
End page
Finish
